packages/web_benchmarks/lib/client.dart

```diff
@@ -122,9 +120,9 @@ void _fallbackToManual(String error) {
         ..allowInlineStyles());
 
   for (final String benchmarkName in _benchmarks.keys) {
-    final html.Element button = html.document.querySelector('#$benchmarkName');
+    final html.Element button = html.document.querySelector('#$benchmarkName')!;
```

Comment:
Why is this safe to force-unwrap?

Reply:
This is safe to unwrap since we are appending these keys L106-L117